src/app/entity-metadata.ts

```diff
@@ -7,17 +7,17 @@ export const entityMetadata: EntityMetadataMap = new Map([
       entityType: ObservabilityEntityType.Api,
       icon: ObservabilityIconType.Api,
       detailPath: (id: string, sourceRoute?: string) => [sourceRoute ?? '', 'endpoint', id],
-      sourceRoutes: ['services']
+      sourceRoutes: ['apis']
```

Comment:
I saw your comment, but unless I'm missing another spot, the only place `endpoints` is used is nested like `service/endpoints`. Rather than being confusing, I'd actually expect them to be consistent since they're showing the same list of entities in different scopes.

Reply:
That's fine. I can go either way. I'll change `apis` to `endpoints`.